backends/hostmem.c

```diff
@@ -320,6 +320,13 @@ size_t host_memory_backend_pagesize(HostMemoryBackend *memdev)
 }
 #endif
 
+void cheri_tag_init(MemoryRegion *mr, uint64_t memory_size);
+__attribute__((weak)) void cheri_tag_init(MemoryRegion *mr,
```

Comment:
Blegh this is all gross but I guess is because backends is built once for all targets so you can't ifdef based on TARGET_FOO?

Reply:
Yeah =/

Reply:
Yeah this is a bit ugly, but seems like the easiest workaround for TARGET_FOO.